doc/source/api_changes.rst

```diff
@@ -23,6 +23,7 @@ Version 0.14
   favor of ``skimage.restoration.denoise_nl_means``.
 - ``skimage.measure.LineModel`` has been removed in favor of
   ``skimage.measure.LineModelND``.
+- In ``skimage.feature.hog`` visualise has been changed to visualize.
```

Comment:
I just fixed test_hog.py and also a reference in another example I found. 

There is an entry in TODO for version 0.16 (if it needs more let me know).